lib/backend/k8s/conversion.go

```diff
@@ -219,6 +219,11 @@ func (c converter) podToWorkloadEndpoint(pod *kapiv1.Pod) (*model.KVPair, error)
 		ipNets = []cnet.IPNet{*ipNet}
 	}
 
+	// Check to see if the Pod has a Node.
+	if len(pod.Spec.NodeName) == 0 {
```

Comment:
If we do hit this - should we error out - the rest of the processing seems a little worrisome if we don't have a node name.